### PR TITLE
[FIX] pos_*: Make point of sale usable on mobile

### PR DESCRIPTION
**Pull request by adr-odoo**
Some UX and UI improvement for POS on smaller screen.

task-2453679

### CODE REVIEW

File: addons/point_of_sale/static/src/js/ChromeWidgets/CashierName.js [hunk 1]
```diff
@@ -7,12 +7,13 @@ odoo.define('point_of_sale.CashierName', function(require) {
     // Previously UsernameWidget
     class CashierName extends PosComponent {
         get username() {
-            const cashier = this.env.pos.get_cashier();
-            if (cashier) {
-                return cashier.name;
-            } else {
-                return '';
-            }
+            const {name} = this.env.pos.get_cashier();
```
Comment (pparidans) [inline addons/point_of_sale/static/src/js/ChromeWidgets/CashierName.js:10]: brackets' spacing `{ name }`

File: addons/point_of_sale/static/src/js/ChromeWidgets/CashierName.js [hunk 2]
```diff
@@ -7,12 +7,13 @@ odoo.define('point_of_sale.CashierName', function(require) {
...
+            return name ? name : '';
+        }
+        get avatar() {
+            const {user_id} = this.env.pos.get_cashier();
```
Comment (pparidans) [inline addons/point_of_sale/static/src/js/ChromeWidgets/CashierName.js:14]: same

File: addons/point_of_sale/static/src/css/pos.css [hunk 3]
```diff
@@ -698,9 +698,10 @@ td {
     flex-grow: 1;
 }
 .pos .actionpad .button {
+    --btnheightsize: 54px;
```
Comment (pparidans) [inline addons/point_of_sale/static/src/css/pos.css:695]: `--btn-height-size` instead ?
(more readable)

File: addons/point_of_sale/static/src/css/pos.css [hunk 4]
```diff
@@ -3457,7 +3457,6 @@ td {
 .receipt-screen .default-view .actions .buttons .button {
     flex: 1;
     border: solid 1px rgb(209, 209, 209);
-    padding: 1rem;
```
Comment (pparidans) [inline addons/point_of_sale/static/src/css/pos.css:3524]: Probably the reason why the "Print receipt" button is too small ?

File: addons/point_of_sale/static/src/xml/Screens/ClientListScreen/ClientListScreen.xml [hunk 5]
```diff
@@ -5,14 +5,18 @@
...
             <div class="screen-content">
                 <div class="top-content">
-                    <div t-if="!state.detailIsShown &amp;&amp; !state.selectedClient" class="button new-customer" role="img" aria-label="Add a customer"
-                          t-on-click="trigger('activate-edit-mode', { isNewClient: true })"
-                          title="Add a customer">
+                    <div class="button back" t-on-click="back">
+                        <i class="fa fa-angle-double-left"/>
                         <t t-if="!env.isMobile">
-                            Create
+                            <span> Discard</span>
```
Comment (pparidans) [inline addons/point_of_sale/static/src/xml/Screens/ClientListScreen/ClientListScreen.xml:11]: why a `<span>` ?

File: addons/point_of_sale/static/src/xml/Screens/ClientListScreen/ClientListScreen.xml [hunk 6]
```diff
@@ -5,14 +5,18 @@
...
                         </t>
-                        <t t-else="">
-                            <i class="fa fa-plus"></i>
+                    </div>
+                    <div t-if="!state.detailIsShown &amp;&amp; !state.selectedClient" class="button new-customer" role="img" aria-label="Add a customer"
+                            t-on-click="trigger('activate-edit-mode', { isNewClient: true })"
+                            title="Add a customer">
+                        <i class="fa fa-plus"/>
+                        <t t-if="!env.isMobile">
+                            <span> Create</span>
```
Comment (pparidans) [inline addons/point_of_sale/static/src/xml/Screens/ClientListScreen/ClientListScreen.xml:19]: Same

File: addons/point_of_sale/__manifest__.py [hunk 7]
```diff
@@ -58,7 +58,6 @@
             'web/static/fonts/fonts.scss',
             'web/static/lib/fontawesome/css/font-awesome.css',
             'point_of_sale/static/src/css/pos.css',
-            'point_of_sale/static/src/css/keyboard.css',
```
Comment (pparidans) [inline addons/point_of_sale/__manifest__.py:61]: Removing files in stable branch? (doing my aab 🤓 )

**Comment by adr-odoo** [comment]
@robodoo rebase-ff

**Review by pparidans — APPROVED** [review]
LGTM 👍

**Comment by pparidans** [comment]
@robodoo r+

**Pull request closed without merging**